File: odd_or_even.py
def odd_or_even(arr):
    """Given a list of numbers, determine whether the sum of its elements is odd or even.
    Give your answer as a string matching "odd" or "even".
    If the input array is empty consider it as: [0] (array with a zero).
    """
    return 'even' if sum(arr) % 2 == 0 or len(arr) == 0 else 'odd'

File: javascript/oddOrEven.js
function oddOrEven(array) {
    /**
     * Given a list of integers, determine whether the sum of its elements is odd or even.
       Give your answer as a string matching "odd" or "even".
       if arr == [] -> 'even'
     */
    return array.length == 0 || array.reduce((prev, cur) => prev + cur) % 2 == 0 ? 'even' : 'odd';
}

console.log(oddOrEven([]))